debug/testlib.py

```diff
@@ -852,6 +854,8 @@ def postMortem(self):
         self.gdb.interrupt()
         self.gdb.command("disassemble", ops=20)
         self.gdb.command("info registers all", ops=100)
+        self.gdb.command("flush regs")
```

Comment:
Should this be before "info registers all"?

Reply:
Probably. I mostly wanted to get the PC which definately needs you to `flush regs`

Reply:
Actually, I think it's correct as-is. GDB already gets the right register values for the current thread when it does the `Ctrl-C`. But it doesn't seem to properly get the right register values for OTHER threads, so when you do `info threads` it doesn't show you their PCs. putting it earlier just wastes register reads and writes.